server/controllers/itemController.js

```diff
@@ -1,28 +1,65 @@
 // Sylvia
 var Items = require('../models/itemModel.js');
+var fs = require('fs-extra');
+var mkdirp = require('mkdirp');
 
```

Comment:
I don't see mkdirp in the package.json, is this an included package or should it be added to the package.json file?